### PR TITLE
feat: add inline data inspector

### PR DESCRIPTION
**Pull request by connor4312**
The old data inspector is preserved, but the new one is the default. It
functions (and looks) like the standard widget in the editor, and
basically just has information parity with the old sidebar inspector.

### CODE REVIEW

**Comment by lramos15** [comment]
I would like to discuss this in the UX sync prior to merging it as I don't love an endianness checkbox and would like others thoughts on this

File: media/editor/vscodeUi.tsx [hunk 1]
```diff
@@ -246,6 +249,12 @@ export const Popover: React.FC<IPopoverProps> = ({ anchor, visible, className, c
 		}
 	}, [visible]);
 
+	useGlobalHandler<MouseEvent>("mousedown", evt => {
+		if (evt.target instanceof Element && !popperElement?.contains(evt.target)) {
+			onClickOutside?.();
```
Comment (lramos15) [inline media/editor/vscodeUi.tsx:254]: Isn't this a required prop when would this be undefined

**Review by lramos15 — CHANGES_REQUESTED** [review]
Left a few comments. I still worry about the maintainability of the hex editor given that this 500 lines of changes doesn't add any comments. Even just general "this clears the tooltip when you click off" or "reveals the data inspector" would be greatly appreciated. Also wonder if we should add some tests for the data inspector given that it can now be in 3 locations we should make sure it works the same in all the various locations. Thanks for the change :)!

File: media/editor/dataInspectorProperties.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	uint32: (dv, le) => dv.getUint32(0, le).toString(),
+	int32: (dv, le) => dv.getInt32(0, le).toString(),
+
+	int64: (dv, le) => dv.getBigInt64(0, le).toString(),
+	uint64: (dv, le) => dv.getBigUint64(0, le).toString(),
+
+	float32: (dv, le) => dv.getFloat32(0, le).toString(),
+	float64: (dv, le) => dv.getFloat64(0, le).toString(),
+
+	"UTF-8": dv => {
```
Comment (lramos15) [inline media/editor/dataInspectorProperties.tsx:30]: Do we still need https://github.com/microsoft/vscode-hexeditor/blob/main/media/data_inspector/byteData.ts then if we do all the data inspector conversions here?

Comment (connor4312) [reply media/editor/dataInspectorProperties.tsx]: Cleaned up, -200 lines!

**Pull request merged**